layouts/team/li.html

```diff
@@ -1,4 +1,5 @@
 <div class="team-member">
     <div class="member-image"><img src="{{.Params.imgSrc}}" alt=""></div>
     <div class="member-name">{{.Params.Title}}</div>
-</div>
\ No newline at end of file
+    <div class="member-email"><a href="mailto:{{.Params.Email}}">{{.Params.Email}}</a></div>
```

Comment:
For custom parameter names in hugo (like `imgSrc`) it can be accsessed in the `.Params` object as is. I believe in this case it should be `.Params.email`